### PR TITLE
Implemented SLA Policy resource

### PR DESCRIPTION
**Pull request by RWejlgaard**

### CODE REVIEW

**Comment by RWejlgaard** [comment]
the build failed because it ran before the sla-api was merged

**Comment by nukosuke** [comment]
I've released v0.6.0 which includes SLA policy support.

**Comment by tamccall** [comment]
Getting the following failure when i run the acceptance tests.

```
--- FAIL: TestAccSLAPolicyExample (0.00s)
    testing.go:635: Step 0 error: /var/folders/3n/2m9krr0j0gbcbw02v755nz240000gn/T/tf-test349118258/main.tf:18,24-25: Unexpected comma after argument; Argument definitions must be separated by newlines, not commas. An argument definition must end with a newline.
```

File: examples/sla_policies.tf [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  all {
+    field    = "type"
+    operator = "is"
+    value    = "incident"
+  }
+
+  policy_metrics {
+    priority = "normal",
+    metric = "first_reply_time",
+    target = 30,
```
Comment (nukosuke) [inline examples/sla_policies.tf]: oh, the error seems to be complaining these lines.

**Comment by tamccall** [comment]
```
--- FAIL: TestAccSLAPolicyExample (0.01s)
    testing.go:635: Step 0 error: config is invalid: Unsupported block type: Blocks of type "policy_metrics" are not expected here.
```

File: zendesk/resource_zendesk_sla_policy.go [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+		},
+		Delete: func(d *schema.ResourceData, i interface{}) error {
+			zd := i.(client.SLAPolicyAPI)
+			return deleteSLAPolicy(d, zd)
+		},
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
```
Comment (tamccall) [inline zendesk/resource_zendesk_sla_policy.go]: policy_metrics are not described in this block

**Pull request merged**